plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -44,7 +44,10 @@ SecRule REQUEST_FILENAME "@endsWith /wp-login.php" \
     t:none,\
     nolog,\
     ctl:ruleRemoveTargetById=931130;ARGS:redirect_to,\
+    ctl:ruleRemoveTargetById=932236;ARGS_NAMES:pwd,\
```

Comment:
I suggest to keep same ARGS togather i.e. move this line up.

Reply:
Wouldn't it be better to order by rule id? it would be easier to spot duplicate ctl actions